python/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "mobilecoin"
+version = "0.1.0"
```

Comment:
Can we update this to stay in sync with the full service repo version?

Reply:
done